driver/config/config.go

```diff
@@ -77,6 +77,8 @@ const (
 	ViperKeyCourierSMSEnabled                                = "courier.sms.enabled"
 	ViperKeyCourierSMSFrom                                   = "courier.sms.from"
 	ViperKeyCourierMessageRetries                            = "courier.message_retries"
+	ViperKeyCourierWorkerPullCount                           = "courier.worker.pull_count"
+	ViperKeyCourierWorkerPullWait                            = "courier.worker.pull_wait"
```

Comment:
Shouldn't this rather be `poll`?

Reply:
Hm I think polling is more when you ask a server / component whether they're ready periodically, and pulling refers to retrieving data and altering state. But really I don't care either way :)

Reply:
Yeah, both would work in this case I think...